pydocteur/pr_status.py

```diff
@@ -17,8 +17,8 @@ def get_checks_statuses_conclusions(pr):
     if not runs:
         logging.info(f"No runs for PR #{pr.number}.")
         return False
-    statuses = [run["status"] for run in runs if run["name"] != "check-title"]
-    conclusions = [run["conclusion"] for run in runs if run["name"] != "check-title"]
```

Comment:
@JulienPalard you removed the check title exclusion so we can think of a better way to handle it for merges ? Or should we just not ignore it?

Reply:
Current state is : `check-title` is not ignored, this mean we can only merge if `check-title` passes (and all other test pass), which is what we want?